news/2021-12-09-winter-fanart-voting.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Winter 2021 Fanart Contest Voting Open"
+date: 2021-12-09 18:00:00 +0000
+---
+
+The festivities of year's end are nearly upon us, so help us pick a few shiny backgrounds for the occasion from a selection of **117** entries!
+
+![](https://assets.ppy.sh/contests/136/header.jpg)
+
+As per usual, the intrepid artists of the osu! community just cannot help themselves when it comes to seasonal fanart, and have absolutely deluged us with a stonking roster of amazing pieces to choose from.
+
+Alas, we only have **12** votes to pass around this year. The torment ever continues.
+
+We're ushering in this fearsome trial with a brief clip showcasing the pieces up for voting to the tunes of the very aptly-named *Winter* by [**3R2 / DJ Mashiro**](https://osu.ppy.sh/beatmaps/artists/187), one of our Featured Artists:
+
+<div align="center">
+    <video width="90%" controls>
+        <source src="https://assets.ppy.sh/contests/136/winter2021-fanart.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+[**CLICK HERE to go straight to the Winter 2021 Fanart Contest voting page and vote today!**](https://osu.ppy.sh/community/contests/136)
```

Comment:
if you capitalise it, then it would seem like the link should only encompass those two words

```suggestion
[**Click here to go straight to the Winter 2021 Fanart Contest voting page and vote today!**](https://osu.ppy.sh/community/contests/136)
```